### PR TITLE
Explore embeddings using tsne

### PR DESCRIPTION
**Pull request by brunosan**
This PR adds a sample notebook to explore the embeddings space using openTSNE locally. Depending on your compute resources it can scale up to the full training set of v0.

It uses Mapbox to pull a RGB context, and also uses -and documents- a few tricks to tSNE such a large corpus with a many dimensions.

This is an example of the output.

Screenshot 2024-01-23 at 10 48 32

### CODE REVIEW

**Review by yellowcap — COMMENTED** [review]
Was not yet successful in running this notebook fully. But here some initial comments.

File: docs/explore_embeddings.ipynb [hunk 1]
```diff
@@ -0,0 +1,462 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import openTSNE\n",
```
[the remark this answers is not in the document]
Comment (weiji14) [reply docs/explore_embeddings.ipynb:252]: Conda doesn't allow optional dependencies yet, but we can maintain an `environment-docs.yml` file for example. Alternatively, we could just put TSNE in the docs GitHub Actions CI if this is only a one-off thing.

**Comment by yellowcap** [comment]
@brunosan I think we should update this to v1 or close. What do you think?

**Comment by brunosan** [comment]
We can re-do. Closing for now

**Pull request closed without merging**